hail/python/hail/experimental/vcf_combiner.py

```diff
@@ -88,14 +88,33 @@ def combine_gvcfs(mts):
     def localize(mt):
         return mt._localize_entries('__entries', '__cols')
 
+    def fix_alleles(alleles):
+        s = hl.set(alleles.map(lambda d: d.ref))
```

Comment:
Since `s` is only being used to find the longest `ref`, the `hl.set` call is redundant. May as well just fold over the unsorted array of refs.